### PR TITLE
Improved sendKeys and added tests

### PR DESCRIPTION
**Pull request by Ad1B3nAr0ya**

### CODE REVIEW

File: src/commons/BrowserUtils.ts [hunk 1]
```diff
@@ -851,7 +856,7 @@ export namespace BrowserUtils {
    * For reporter log propose in order to log what keys were send
    * @param keysToSend key/keys of SpecialKeys type
    */
-  function getKeyNames(keysToSend: SpecialKeys | SpecialKeys[]): string {
+  function getKeyNames(keysToSend: SpecialKeys | SpecialKeys[] | string[]): string {
```
Comment (FelixZilber) [inline src/commons/BrowserUtils.ts]: does not looks that this function have been called with string[]

File: test/specs/SendKeysSpec.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    BrowserUtils.sendKeys(text.split(''));
+    assert.equal($(TEXT_INPUT).getValue(), text);
+  });
+
+  it('Add text', () => {
+    $(TEXT_INPUT).clearValue();
+    $(TEXT_INPUT).click();
+    BrowserUtils.sendKeys(text.split(''));
+    $(TEXT_INPUT).click();
+    BrowserUtils.sendKeys(text.split(''));
```
Comment (FelixZilber) [inline test/specs/SendKeysSpec.ts]: why do you have double input on every test?

File: src/commons/BrowserUtils.ts [hunk 3]
```diff
@@ -777,10 +777,15 @@ export namespace BrowserUtils {
   /**
    * Send a sequence of key strokes to the active element
    * it can be single key or an array of keys
-   * @param keysToSend key or array of keys to send
+   * @param keysToSend key, array of keys or string array (chars) to send
    */
-  export function sendKeys(keysToSend: SpecialKeys | SpecialKeys[]): void {
-    Reporter.debug(`Sending Keys ${getKeyNames(keysToSend)}`);
+  export function sendKeys(keysToSend: SpecialKeys | SpecialKeys[] | string[]): void {
```
Comment (FelixZilber) [inline src/commons/BrowserUtils.ts]: why do you use string[] ? what benefits does it have over string ?

File: test/specs/SendKeysSpec.ts (hunk 2 above)
Comment (Ad1B3nAr0ya) [reply test/specs/SendKeysSpec.ts]: To check add text instead of new text list setValue

File: src/commons/BrowserUtils.ts (hunk 3 above)
Comment (Ad1B3nAr0ya) [reply src/commons/BrowserUtils.ts]: Don't know. the example presented as a solution was done using string[]

File: src/commons/BrowserUtils.ts (hunk 1 above)
Comment (Ad1B3nAr0ya) [reply src/commons/BrowserUtils.ts]: obligated by rules of ts.

File: test/specs/SendKeysSpec.ts (hunk 2 above)
Comment (FelixZilber) [reply test/specs/SendKeysSpec.ts]: just remove second click

Comment (Ad1B3nAr0ya) [reply test/specs/SendKeysSpec.ts]: fixed

File: src/commons/BrowserUtils.ts (hunk 3 above)
Comment (Ad1B3nAr0ya) [reply src/commons/BrowserUtils.ts]: Changed type to string and splice inside sendKeys to avoid saved word such as 'Enter', 'F5' etc'

**Pull request merged**